app/models/report.rb

```diff
@@ -0,0 +1,10 @@
+class Report
+  attr_reader(:year, :month, :cause, :event)
+
+  def initialize(params)
+    @year = params[:date][:year]
+    @month = params[:date][:month]
+    @cause = params[:cause_id]
```

Comment:
We shouldn't name these `cause` and `event` if they don't hold causes or events. I think what we mean to do here is to actually find these records in the initializer?